### PR TITLE
Add Meeting Minutes: 2018-03-08

### LINKED ISSUE #261: Node.js Foundation Community Committee Meeting 2018-03-08
## Time

**UTC Thu 08-Mar-2018 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 08-Mar-2018 09:00 (09:00 AM) |
| US / Mountain | Thu 08-Mar-2018 10:00 (10:00 AM) |
| US / Central  | Thu 08-Mar-2018 11:00 (11:00 AM) |
| US / Eastern  | Thu 08-Mar-2018 12:00 (12:00 PM) |
| London        | Thu 08-Mar-2018 17:00 (05:00 PM) |
| Amsterdam     | Thu 08-Mar-2018 18:00 (06:00 PM) |
| Moscow        | Thu 08-Mar-2018 20:00 (08:00 PM) |
| Chennai       | Thu 08-Mar-2018 22:30 (10:30 PM) |
| Hangzhou      | Fri 09-Mar-2018 01:00 (01:00 AM) |
| Tokyo         | Fri 09-Mar-2018 02:00 (02:00 AM) |
| Sydney        | Fri 09-Mar-2018 04:00 (04:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Community%20Committee+Meeting+2018-03-08&iso=20180308T17
* or http://www.wolframalpha.com/input/?i=05PM+UTC%2C+Mar+08%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1rFQZE_KuNiFvFfcAamCT978bvJ9K3t9EeoLCcCdtTSE/edit>

## Agenda

Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

* Initiative Review

### nodejs/summit

* Collaborator Summit Berlin 2018 - May 31st, June 1st 2018 [#60](https://github.com/nodejs/summit/issues/60)

### nodejs/community-committee
* Diversity/Inclusivity Initiative? [#260](https://github.com/nodejs/community-committee/issues/260)
* Re-creating official secretary role [#241](https://github.com/nodejs/community-committee/issues/241)
* Automate Meeting Participant History Document [#267](https://github.com/nodejs/community-committee/issues/267)
* Current Strategic Initiatives Review [#222](https://github.com/nodejs/community-committee/issues/222)

### nodejs/mentorship

* communication channel for the matched mentor/mentee 's first meeting [#11](https://github.com/nodejs/mentorship/issues/11)

## Invited

* CommComm Members: @nodejs/community-committee

### Observers
* @therebelrobot (Oz Haven - observer)
* @ParidelPooya (Pooya Paridel - observer)
* @rykerrumsey (Ryker Rumsey - observer)
* @baez (Behzad Karim - observer)
* @BinarySo1o (Therese Stirling - observer)
* @amiller-gh (Adam Miller - observer)
* @yosuke-furukawa (Yosuke Furukawa - observer)
* @Maurice-Hayward (Maurice Hayward - observer)
* @mikehostetler (Mike Hostetler - observer)
* @sarahkconway (Sarah Conway - observer)
* @Tiriel (Benjamin Zaslavsky - observer)
* @DiegoZoracKy (Diego ZoracKy - observer)
* @ZibbyKeaton (Zibby Keaton - observer)
* @Bamieh (Ahmad Bamieh - observer)
* @pchrysa (Chrysa - observer)
* @foadlab (observer)
* @codeekage (Abraham Agiri - observer)
* @Voskan (Voskan Voskanyan - observer)
* @Tiriel (observer)
* @WaleedAshraf (observer)
* @maddhruv (observer)
* inidaname (Hassan Sani - observer)

## Notes

The agenda comes from issues labelled with `cc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: Will be added a few minutes before meeting starts
* For those who just want to watch: 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

### PR DESCRIPTION
**Pull request by JemBijoux**
Add minutes for today's meeting.

Closes #261

### CODE REVIEW

**Comment by codeekage** [comment]
@JemBijoux  I was not added as present at the meeting...

File: meetings/2018-03-08.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+- Gregor Martynus (@gr2m)
+- Manil (@chowdhurian)
+- Dan Shaw (@dshaw)
+- Michael Dawson (@mhdawson)
+- William Kapke (@williamkapke)
+- Adam Miller (@amiller-gh)
+- Benjamin Zaslavsky (@Tiriel)
+- Zibby Keaton (observer)
+- Ahmad (@Bamieh observer)
+
```
Comment (WaleedAshraf) [inline meetings/2018-03-08.md]: Waleed Ashraf (@waleedashraf observer)

**Comment by WaleedAshraf** [comment]
Yes. I missed couple of minutes in beginning.
What I have realised now is that, zoom ID is available to join even before meeting has started? Right?

So instead of joining right on time and than looking for handsfree and best wifi connection, it can be done earlier. (This always takes me couple of minutes)

File: meetings/2018-03-08.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+- One of the first initiatives, but repo may be archived because of inactivity
+- largely driven by a previous observer but they felt they didn't get the steam needed to move it forward
+- Tierney offered to help promote that, and put word out for interest and additional supporters
+- An issue will be created in next day or so
+- Seems like it could be pretty tied to the Nodejs africa work: some contributors there really excited and might match up well
+
+### nodejs/summit
+
+* Collaborator Summit Berlin 2018 - May 31st, June 1st 2018 [#60]([https://github.com/nodejs/summit/issues/60](https://github.com/nodejs/summit/issues/60))
+- Dan and Tracy organizing
```
Comment (oe) [inline meetings/2018-03-08.md]: me tooooo

**Comment by JemBijoux** [comment]
👍 Let me know if we need anything else changed here.

**Comment by bnb** [comment]
Merging 👍

**Pull request merged**